Gemfile

```diff
@@ -3,11 +3,5 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in cache_store_redis.gemspec
 gemspec
 
-platforms :ruby do
-  gem 'oj', '2.15.0'
-  gem 'pry'
-end
-
-platforms :jruby do
-  gem 'pry-debugger-jruby'
-end
\ No newline at end of file
+gem 'oj', '2.15.0'
```

Comment:
this looks old?
The service uses `(3.6.10)`
https://github.com/Sage/s1_event_gateway_service/blob/master/src/Gemfile.lock#L80